spec/recipes/default_spec.rb

```diff
@@ -24,9 +24,12 @@
   let(:crt_file) { 'logs-01.loggly.com_sha12.crt' }
   let(:crt_file_path) { "#{crt_path}/#{crt_file}" }
   let(:crt_remote_resource) { 'download loggly.com cert' }
+  let(:rsyslog_group) { 'syslog' }
```

Comment:
The default changed to `root` :point_up:. I'm not sure if that should be the case here as well.